common-tasks/usb.md

```diff
@@ -147,162 +297,6 @@ steps:
-     (recommended: `sys-usb`, red). If you need to attach a networking device,
-     it might make sense to create a NetVM. If not, an AppVM might make more
-     sense. (The default `sys-usb` is a NetVM.)
- 3.  In the qube's settings, go to the "Devices" tab. Find the USB controller
-     that you identified in step 1 in the "Available" list. Move it to the
-     "Selected" list.
-
-     **Caution:** By assigning a USB controller to a USB qube, it will no longer
-     be available to dom0. This can make your system unusable if, for example,
-     you have only one USB controller, and you are running Qubes off of a USB
-     drive.
-
- 4.  Click "OK." Restart the qube.
- 5.  Recommended: Check the box on the "Basic" tab which says "Start VM
-     automatically on boot." (This will help to mitigate attacks in which
-     someone forces your system to reboot, then plugs in a malicious USB
-     device.)
-
-If the USB qube will not start, see [here][faq-usbvm].
-
-
-Removing a USB qube
--------------------
-
-**Warning:** This procedure will result in your USB controller(s) being attached
-directly to dom0.
-
-1. Shut down the USB qube.
-2. In Qubes Manager, right-click on the USB qube and select "Remove VM."
-3. Open the file `/etc/default/grub` in dom0.
-4. Find the line(s) that begins with `GRUB_CMDLINE_LINUX`.
-5. If `rd.qubes.hide_all_usb` appears anywhere in those lines, remove it.
-6. Save and close the file.
-7. Run the command `grub2-mkconfig -o /boot/grub2/grub.cfg` in dom0.
-8. Reboot.
-
-
-How to hide all USB controllers from dom0
------------------------------------------
-
-Even if you create a USB qube, there will be a brief period of time during the
-boot process during which dom0 will be exposed to your USB controllers (and any
-attached devices). This is a potential security risk, since even brief exposure
-to a malicious USB device could result in dom0 being compromised. There are two
-approaches to this problem:
-
-1. Physically disconnect all USB devices whenever you reboot the host.
-2. Hide (i.e., blacklist) all USB controllers from dom0.
-
-**Warning:** If you use a USB [AEM] device, do not use the second option. Using
-a USB AEM device requires dom0 to have access to the USB controller to which
-your USB AEM device is attached. If dom0 cannot read your USB AEM device, AEM
-will hang.
-
-The procedure to hide all USB controllers from dom0 is as follows:
-
-1. Open the file `/etc/default/grub` in dom0.
-2. Find the line that begins with `GRUB_CMDLINE_LINUX`.
-3. Add `rd.qubes.hide_all_usb` to that line.
-4. Save and close the file.
-5. Run the command `grub2-mkconfig -o /boot/grub2/grub.cfg` in dom0.
-6. Reboot.
-
-(Note: Beginning with R3.2, `rd.qubes.hide_all_usb` is set automatically if you
-opt to create a USB qube during installation. This also occurs automatically if
-you choose to [create a USB qube] using the `qubesctl` method, which is the
-first pair of steps in the linked section.)
-
-Supported USB device types
```

Comment:
I removed the following part because it directly contradicts the USB passthrough guide that states that any USB device can be passed through: 

> "-As of Qubes R3.1, it is possible to attach:
> -
> - * USB mice
> - * USB keyboards (see below)
> - * USB block devices (such as USB mass storage devices)
> -   * When attaching one of these, you should get a notification about the
> -     new device, then you should be able to attach it to a qube in Qubes VM
> -     Manager.
> -
> -Other devices, such as USB webcams, will also work, but they will be
> -accessible only from the USB qube itself, as explained above."